mozilla/fx-private-relay
# Restore MaxDiff and NPS micro-survey in place of VPN Promo

The VPN promo has very little effect so far. We want to restore the MaxDiff survey recruitment banner (for Premium Relay feature direction) and the NPS micro-survey (for Relay user satisfaction OKR) banner instead.

Testing instructions:
1. Set a new env var `INCLUDE_VPN_BANNER` to `False` (not strictly required, `False` is the default value)
2. Set 2 more env vars:
   * `RECRUITMENT_BANNER_LINK=https://survey.alchemer.com/s3/6448870/Relay-
Features-MaxDiff-Q3-21`
   *  `RECRUITMENT_BANNER_TEXT="How can we improve Relay for you? Let us know."`
3. Restart the server
4. Clear all cookies for 127.0.0.1
5. Go to 127.0.0.1:8000 and sign in
   * You should see the "How can we improve Relay for you? Let us know." banner at the top.
6. Click the banner link to go to the survey
   * You don't have to fill out the survey
7. Go back to 127.0.0.1:8000 and reload
   * The "improve Relay" banner should be gone
   * You should now see the "On a scale from 1-10, how likely are you to recommend ⁨Relay⁩ to a friend or colleague?" micro-survey banner at the top
8. Click an option in the micro-survey
   * It should go away
9. Reload the page
   * You should see no more banners anymore
10. Close Firefox and re-open it
11. Go back to 127.0.0.1:8000
   * You should still not see any banners
